### PR TITLE
Edited README

### PR DESCRIPTION
**Pull request by guillermooo**
- Tried to use fewer words where possible
- Favor active voice where possible

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
+We started this project to fill the gaps in the Sublime Text
+[official documentation][off-docs]. You can visit this guide at
+<http://docs.sublimetext.info/>. It is hosted via [ReadTheDocs][] and built
+using [Sphinx][].
 
-Following our [fundraiser on BountySource][fundraiser] during July-August 2014
-we are currently in the process of revising all topics and adding more topics
-in general. You can follow our progress on our public [Trello board][trello].
+Following our [fundraiser on BountySource][fundraiser] during July-August 2014,
+we've started revising all topics and adding new ones. You can follow our
```
Comment (guillermooo) [inline README.md:10]: perhaps rather 'begun' to avoid repeating 'started' so often.

File: README.md [hunk 2]
```diff
@@ -1,50 +1,51 @@
...
-settings and a build system that builds the HTML docs using [Sphinx][].
+This repository includes a `.sublime-project` with predefined settings and a
+build system that builds the HTML docs using [Sphinx][].
 
 
 ### Building (HTML Preview)
 
-In order to build and preview the docs you need Sphinx, which in return
-requires **Python 2.7**. Python 3+ does not work.
+In order to build and preview the docs you'll need Sphinx, which in return
```
Comment (guillermooo) [inline README.md]: return => turn

File: README.md (hunk 1 above)
Comment (FichteFoll) [inline README.md:10]: It's used exactly twice in the entire file, but both occurences are at the beginning. Idc.

File: README.md [hunk 3]
```diff
@@ -1,50 +1,51 @@
...
 
 *PayPal/Gittip/Pledgie be here*
 
 
 ## Contributing
 
-You are free to submit any form of issue on our [issues][] page and we are very
-open to pull requests. Your PR will be reviewed and might not be accepted.
-**Please follow our (style) guidelines in
+We accept error reports and feature requests via our [issue tracker][] and
```
Comment (FichteFoll) [inline README.md]: What kind of "feature" would you request in a documentation?

Comment (guillermooo) [inline README.md]: Good point. Maybe something along the lines of 'improvements' or 'suggestions for new content'.

File: README.md (hunk 1 above)
Comment (guillermooo) [inline README.md:10]: Nevermind, it isn't that important.

**Pull request merged**